redbot/core/bank.py

```diff
@@ -524,6 +544,14 @@ async def bank_prune(bot: Red, guild: discord.Guild = None, user_id: int = None)
 
     """
 
+    print(f"{guild} | {user_id}")
+    if guild is None and user_id is None:
+        scope = 1  # prune global
+    if guild is not None and user_id is None:
+        scope = 2  # prune server
+    else:
+        scope = 3  # prune user
```

Comment:
Forgot to hit submit on the comment I was writing here....

Doing it like this loses the ability to know what guild was being pruned. Unless there is some reason to differentiate between specifically pruning a member and a single member happening to be the only invalid member from a bulk prune, the signature of this event object would likely be better as:
`guild: Optional[discord.Guild], pruned_user_ids: List[int]`